packages/template-retail-react-app/app/commerce-api/hooks/useEinstein.js

```diff
@@ -19,6 +19,24 @@ const useEinstein = () => {
         async sendViewProduct(...args) {
             return api.einstein.sendViewProduct(...args)
         },
+        async sendViewSearch(...args) {
+            return api.einstein.sendViewSearch(...args)
+        },
+        async sendClickSearch(...args) {
+            return api.einstein.sendClickSearch(...args)
+        },
+        async sendViewCategory(...args) {
+            return api.einstein.sendViewCategory(...args)
+        },
+        async sendClickCategory(...args) {
+            return api.einstein.sendClickCategory(...args)
+        },
+        async sendViewPage(...args) {
+            return api.einstein.sendViewPage(...args)
+        },
+        async sendBeginCheckout(...args) {
+            return api.einstein.sendBeginCheckout(...args)
+        },
```

Comment:
Again, if you squint, all these lines look the same. This is probably something to DRY up at some point. If you are feeling proactive, start a 1-pager on how we can look into refactoring this Einstein stuff.